src/components/RestaurantDetails.vue

```diff
@@ -36,8 +36,7 @@
       <h4>Linki</h4>
       <ul>
         <li>
-          <a :href="restaurant.url">{{ restaurant.name }}</a
-          ><br />
+          <a :href="restaurant.url">{{ restaurant.name }}</a><br />
```

Comment:
Please make separate commits for separate issues, especially if they're in separate files (it's not a problem to first commit one file and then the other one).

Some programmers even claim that style changes should have their own PRs but I'm not as big of a fundamentalist.